### ISSUE TITLE
Recorder disable

### ISSUE THREAD

**Issue by pepe59** [thomasmauerer/hassio-addons#66]
It could be added to the add-in to start the recorder.disable service before the backup and the recorder.enable service after the backup is complete. I think this would help eliminate the db corruption issue.

**Comment by cryptedx** [comment]
Should be fixed with MariaDB Version 2.4.0
=> Add lock capabilities during snapshot

This is my log while Samba Backup is running:

```
[02:44:09] INFO: Lock tables using mariadb client...
[02:44:09] INFO: MariaDB tables locked.
[02:52:21] INFO: MariaDB tables unlocked.
```

**Comment by thomasmauerer** [comment]
Finally I had a chance now to create and restore a backup including MariaDB. It seems that the locking mechanism introduced in v2.4.0 works as expected. I could not see any issue.

I'm closing this now. Thanks @cryptedx for the hint.
